Makefile

```diff
@@ -307,7 +307,7 @@ mocks: ## Generate mocks for the project
 	mockgen -destination=mocks/k8sclient/cm_informer.go -package=mockk8sclient k8s.io/client-go/informers/core/v1 ConfigMapInformer
 	mockgen -destination=mocks/k8sclient/secret_informer.go -package=mockk8sclient k8s.io/client-go/informers/core/v1 SecretInformer
 
-GOTESTPKGS = $(shell go list ./... | grep -v /mocks)
+GOTESTPKGS = $(shell go list ./... | grep -v /mocks | grep -v /templates)
```

Comment:
do we need tee instead of pipe for 2 greps to work or is templates inside /mocks